### ISSUE TITLE
can it compile to windows also?

### ISSUE THREAD

**Issue by talregev** [bcharrow/matlab_rosbag#13]
Thanks for the work!

Tal,

**Comment by reinzor** [comment]
I;m also interested in this

**Comment by ibadr** [comment]
Yes, it can be compiled for Windows. I just tried it and it seems to work. I created [this fork](https://github.com/ibadr/matlab_rosbag/releases/) to deliver MEX files for Windows. The challenging part was to compile the needed ROS packages under Windows.

**Comment by talregev** [comment]
it working!
thx @ibadr  :+1:

**Comment by sahiedu** [comment]
Which version of MATLAB should I use in order to use 'matlab_rosbag-0.5.0-win64'? I tried with a recent version and I got many errors.

**Comment by chien-lin-tseng** [comment]
thanks @ibadr 
it really works on my win10 laptop!
